governance/tag-attribution.md

```diff
@@ -0,0 +1,68 @@
+# TAG Attribution Guidelines
+
+[Attribution] is the act of ascribing a work to a particular author.
+
+The TAG may be attributed with authorship of a work following a robust
+community review and approval process. Depending on the document category, a
+work will require different review processes before attribution is possible.
+
+Below is a list of works categories, and what will enable it to be attributed
+as a TAG publication.
+
+1. Publications (whitepapers, etc)
+
+- The document must follow all guidelines, protocols, and processes outlined
+    in the [publications] directory.
+
+1. Presentations (conference talks, webinars, etc)
```

Comment:
Does this mean that a presentation would have to be approved every time someone gives a shout out to TAG Security? For example in the lead-up to CloudNativeSecurityCon I gave several explanations of what the TAG does, etc, which I would not have been able to do if it required an approval process.

Reply:
We could modify the second point below to use a type of "lazy consensus" language, but I imagine that yes— we would want to have some reasonable ability for leads to ensure that content is up to date before it is presented to the public as representative of the group.

For example, if @sublimino does a presentation about the TAG (as he often does), we should have a process that he can easily follow to help him share the latest from the group.

But if @sublimino was doing a presentation about his experience in the TAG (not representing the group) then it wouldn't benefit from the additional support layer.